DuckDuckGoTests/HomePageConfigurationTests.swift

```diff
@@ -24,81 +24,68 @@ import XCTest
 
 class HomePageConfigurationTests: XCTestCase {
 
-    struct Test {
-
-        let layout: HomePageLayout
-        let favorites: Bool
-        let links: [Link]
-        let expected: [HomePageConfiguration.Component]
-
+    func testLayoutCentered() {
```

Comment:
Convention that we try to use for test names is `testWhen<condition>Then<expectation>`, e.g. `testWhenCenteredAndFavoritesDisabledThenFixedCenteredIsUsed`